app/views/leakdetection/LeakDetectionRulesPage.scala.html

```diff
@@ -25,60 +25,74 @@
-            <h1>Leak Detection Rules</h1>
-        </header>
+@standard_layout(s"Leak detection rules", active = "health") {
+    <h1 class="page-heading mt-4">Leak Detection Rules</h1>
 
-        @if(summaries.nonEmpty) {
-            <div id="rules-list">
-                <table class="table table-striped">
-                    <thead>
-                        <tr>
-                            <th></th>
-                            <th class="col-xs-3"><button class="sort no-border" data-sort="rule-id">Rule id</button></th>
-                            <th class="col-xs-6"><button class="sort no-border" data-sort="description">Description</button></th>
-                            <th class="col-xs-1 text-center"><button class="sort no-border" data-sort="repo-count">Repositories</button></th>
-                            <th class="col-xs-1 text-center"><button class="sort no-border" data-sort="excluded">Exemptions</button></th>
-                            <th class="col-xs-1 text-center"><button class="sort no-border" data-sort="unresolved">Violations</button></th>
-                        </tr>
-                    </thead>
-                    <tbody class="list">
+    @if(summaries.nonEmpty) {
+        <div id="rules-list">
+            <table class="table table-striped">
+                <thead>
+                    <tr>
+                        <th></th>
+                        <th class="col-xs-3"><button class="sort no-border fw-bold" data-sort="rule-id">Rule id</button></th>
+                        <th class="col-xs-6"><button class="sort no-border fw-bold" data-sort="description">Description</button></th>
+                        <th class="col-xs-1 text-center"><button class="sort no-border fw-bold" data-sort="repo-count">Repositories</button></th>
+                        <th class="col-xs-1 text-center"><button class="sort no-border fw-bold" data-sort="excluded">Exemptions</button></th>
+                        <th class="col-xs-1 text-center"><button class="sort no-border fw-bold" data-sort="unresolved">Violations</button></th>
+                    </tr>
+                </thead>
+                <tbody class="list">
                     @summaries.map(leakDetectionRuleSummary)
-                    </tbody>
-                </table>
-            </div>
+                </tbody>
+            </table>
+        </div>
 
-            <script @CSPNonce.attr>
-                var options = {valueNames: ['rule-id', 'description', 'first-scanned', 'last-scanned', 'repo-count', 'excluded', 'unresolved']};
-                rulesList = new List('rules-list', options);
-            </script>
-        }
+        <script @CSPNonce.attr>
+            let options = {valueNames: ['rule-id', 'description', 'first-scanned', 'last-scanned', 'repo-count', 'excluded', 'unresolved']};
+            rulesList = new List('rules-list', options);
+
+            document.querySelectorAll('[data-bs-toggle="collapse"]').forEach( function(el) {
```

Comment:
I think we are using this in three different places, is it worth moving it to the catalogue-frontend.js file?